saleor/attribute/__init__.py

```diff
@@ -29,7 +29,7 @@ class AttributeInputType:
     ]
 
     # list of the input types that can be used in variant selection
-    ALLOWED_IN_VARIANT_SELECTION = [DROPDOWN, BOOLEAN, SWATCH]
+    # ALLOWED_IN_VARIANT_SELECTION = [DROPDOWN, BOOLEAN, SWATCH]
```

Comment:
Just to make it clear - to this point Saleor supported these three input types as variant selection, but we should also support `NUMERIC`.

Reply:
Np, I have restored that list (add `NUMERIC`) value, and I have used it in mutation to validate data.